verify.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'enumerable'
+
+class List
+  @list = []
+  include Enumberable
+
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&list)
+    @list.each(&list)
+  end
+end
```

Comment:
- [  ] Please as in the requirements #each should yields successive members of @list and use the MyEnumerable module to consider using the yield keyword to return each number in the list at each time currently, your function returns an Enumerator.
![image](https://user-images.githubusercontent.com/74744787/222912465-f949df92-ac54-461c-9236-2dd8c63e4109.png)